python/lsst/rapid/analysis/utils.py

```diff
@@ -217,3 +221,47 @@ def checkRubinTvExternalPackages(exitIfNotFound=True, logger=None):
 
     if exitIfNotFound and (not hasGoogleStorage or not hasEfdClient):
         exit()
+
+
+def getExpPositionOffset(exp1, exp2):
+    # need the exps if we want altAz because we need the observation times
+    wcs1 = exp1.getWcs()
+    wcs2 = exp2.getWcs()
+    p1 = wcs1.getSkyOrigin()
+    p2 = wcs2.getSkyOrigin()
+
+    vi1 = exp1.getInfo().getVisitInfo()
+    vi2 = exp2.getInfo().getVisitInfo()
+
+    # AltAz via astropy
```

Comment:
Not a fan of this code existing in two places. Consider a helper function that takes the `Exposure` and returns the information you want. It would half the number of lines.